EXERCISES/exercise20.js

```diff
@@ -0,0 +1,53 @@
+/**Crie um programa para informar quais e quantas notas são necessárias para entregar o mínimo de cédulas
+para um determinado valor informado pelo usuário considerando notas de R$ 100, R$ 50, R$ 10 e R$ 5 e R$ 1.
+Seu programa deve mostrar apenas as notas utilizadas. Por exemplo, ao solicitar R$18, o programa deve
+informar apenas a seguinte informação (note que não foram exibidas informações sobre as demais cédulas): 1
+nota(s) de R$ 10. 1 nota(s) de R$ 5. 3 nota(s) de R$ 1. */
+
+function calculaCedulas(x){
+  const valor = isNaN(x) ? 0 : x;
```

Comment:
tenta resolver isso aqui com recursão, OOOOOU, cria um array com as cédulas, e anda por esse array dividindo pelo valor dele e calculando o resto, assim não tem que ficar tantos blocos de código, algo do tipo:
let valor = valorInformado
const cedulas = [100, 50, 20, 10, 5, 2, 1]
for (cedula in cedulas) {
  //logica aqui
}